src/app/components/NavigationBar/NavigationBar.tsx

```diff
@@ -178,17 +178,19 @@ const NavigationBar = ({
 
 	const scroll = useScroll();
 
+	const isVariantFull = variant === "full";
+
 	return (
 		<NavWrapper
 			aria-labelledby="main menu"
-			noBorder={noBorder !== undefined ? noBorder : variant !== "full"}
+			noBorder={noBorder === undefined ? !isVariantFull : noBorder}
 			noShadow={noShadow}
 			scroll={scroll}
 		>
 			<div className="flex relative h-21">
-				{variant === "full" && <BackButton className="flex w-12" disabled={isBackDisabled} />}
+				{isVariantFull && <BackButton className="flex w-12" disabled={isBackDisabled} />}
 
-				<div className={`flex flex-1 px-8 ${variant !== "full" ? "ml-12" : ""}`}>
+				<div className={`flex flex-1 px-8 ${isVariantFull ? "" : "ml-12"}`}>
```

Comment:
```suggestion
				<div className={cn("flex flex-1 px-8", { "ml-12": !isVariantFull })}>
```

we can use the `classnames` package here to avoid the trailing space if `isVariantFull === true`.